src/Symfony/Bundle/SecurityBundle/Tests/DependencyInjection/XmlCompleteConfigurationTest.php

```diff
@@ -17,6 +17,11 @@
 
 class XmlCompleteConfigurationTest extends CompleteConfigurationTestCase
 {
+    public function testFirewallPatterns()
+    {
+        $this->markTestSkipped('This features is not supported in XML.');
```

Comment:
Is there a way we could support this in XML? 🤔

Reply:
I'm really not an XML expert (and I don't like to support something nobody use 😁) but I fail to see how we can do that because the current schema is : 
```
<firewall name="simple" pattern="/login" security="false" />
```

Reply:
I see how this complicates things, thanks.